packages/delivery-fetch/delivery.js

```diff
@@ -1,18 +1,44 @@
 import payload from '@bugsnag/core/lib/json-payload'
 
-const delivery = (client, fetch = global.fetch) => ({
+function getIntegrityHeaderValue (sendPayloadChecksums, windowOrWorkerGlobalScope, requestBody, headers) {
+  if (sendPayloadChecksums && windowOrWorkerGlobalScope.isSecureContext && windowOrWorkerGlobalScope.crypto && windowOrWorkerGlobalScope.crypto.subtle && windowOrWorkerGlobalScope.crypto.subtle.digest && typeof TextEncoder === 'function') {
```

Comment:
see https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto and https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts